Filelist

```diff
@@ -219,6 +219,7 @@ SRC_ALL =	\
 		src/testdir/silent.wav \
 		src/testdir/popupbounce.vim \
 		src/testdir/crash/* \
+		src/testdir/ru_RU/LC_MESSAGES/__PACKAGE__.mo \
```

Comment:
I think we also need the __PACKAGE__.po file

Reply:
Maybe, but I don't know where it's gone :-/